appveyor.yml

```diff
@@ -1,15 +1,20 @@
 environment:
   matrix:
+    - nodejs_version: "8.11.3"
     - nodejs_version: "8"
     - nodejs_version: "10"
+cache:
+ - "%LOCALAPPDATA%\\Yarn"
 install:
-  # get the latest version of Node.js in a release series
   - ps: Install-Product node $env:nodejs_version
-  # output useful info for debugging
+  - ps: Start-FileDownload 'https://github.com/yarnpkg/yarn/releases/download/v1.12.3/yarn-1.12.3.msi'
+  - cmd /c start /wait msiexec.exe /i yarn-1.12.3.msi /quiet /qn /norestart
```

Comment:
Make sure the version of yarn being used in CI is lowest version we support per `"engines"` of embark/-ui's `package.json`.